webclient/app/src/app/commons/entityDialog/Entity.js

```diff
@@ -21,7 +21,7 @@ const newEntity : EntityDataType = {
     }
 };
 
-const nullEntity : EntityDataType = {
```

Comment:
This is not valid JS - this is Typescript

Reply:
bugticket ab#152